core/src/main/java/org/commonjava/indy/core/content/DefaultContentManager.java

```diff
@@ -311,7 +311,16 @@ private boolean checkListingMask( final ArtifactStore store, final String path )
 
         AbstractRepository repo = (AbstractRepository) store;
         Set<String> maskPatterns = repo.getPathMaskPatterns();
-        logger.debug( "Checking mask in: {}, type: {}, patterns: {}", repo.getName(), repo.getKey().getType(), maskPatterns );
+
+        if ( logger.isTraceEnabled() )
+        {
+            logger.trace( "Checking mask in: {}, type: {}, patterns: {}", repo.getName(), repo.getKey().getType(),
+                          maskPatterns );
+        }
+        else
+        {
+            logger.debug( "Checking mask in: {}, type: {}", repo.getName(), repo.getKey().getType() );
+        }
```

Comment:
@ligangty @yma88 It seems Yma added allowDeletion() in Galley Location interface and it breaks Indy. Should we set a default value for this method (only java 8 support it) or fixing Indy impl classes?